Library/Homebrew/test/livecheck/strategy/sparkle_spec.rb

```diff
@@ -147,29 +285,51 @@
     end
   end
 
+  # The Sparkle Strategy only sorts items in `#versions_from_content`.
+  # This block thus expects things to remain in the order they're listed
+  # in the `xml` and `items` data structures.
```

Comment:
```suggestion
```

At least to me, I don't think there's value in explaining that `#items_from_content` doesn't sort items. I think it's reasonable to expect that the items will simply be in the order they appear in the appcast. That said, I think we should omit this comment and simply explain how `#versions_from_content` sorts items before the related tests (as that's more likely to be unexpected behavior).